src/node_url.cc

```diff
@@ -62,6 +62,7 @@ void SetArgs(Environment* env, Local<Value> argv[12], const ada::result& url) {
   argv[10] = Utf8String(isolate, url->get_hash());
   argv[11] = Boolean::New(isolate, url->host.has_value());
   argv[12] = Boolean::New(isolate, url->has_opaque_path);
+  argv[13] = Boolean::New(isolate, url->fragment.has_value());
```

Comment:
Please update the `Local<Value> argv[12]` argument to reflect this change. Even better, use the advice in #46541 and change this to `Local<Value> (*argv)[14]`.